backend/testfiles/execution/internal.tests

```diff
@@ -3,9 +3,23 @@
 // ---------------
 // Misc
 // ---------------
+[tests.misc]
+
+[test.correct number of tables]
 Dict.size_v0 DarkInternal.getAndLogTableSizes_v0 = 24
+
+[test.allFunctions has many functions]
 (List.length_v0 DarkInternal.allFunctions_v0 > 290) = true
 
+[test.server build hash]
+(match DarkInternal.serverBuildHash_v0 with
+ // in local dev, the value is "dev"
+ | "dev" -> true
+ // in ci, "circleci"
+ | "circleci" -> true
+ // otherwise it's the first 7 chars of the git hash
+ | hash -> (String.length hash) = 7)
```

Comment:
This test feels a little bit silly currently. I was surprised circleci didn't have context of the git hash.